### ISSUE TITLE
Cannot read property 'version' of undefined when rendering on the server

### ISSUE THREAD

**Issue by benjamin-albert** [VadimDez/ng2-pdf-viewer#371]
##### Bug Report or Feature Request (mark with an `x`)
```
- [x] Regression (a behavior that used to work and stopped working in a new release)
- [ ] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```

After updating to `5.1.1` we end up with this error in our server logs when rendering the `pdf-viewer` component in the server.
```
ERROR TypeError: Cannot read property 'version' of undefined 
     at new PdfViewerComponent (/app/node_modules/ng2-pdf-viewer/bundles/ng2-pdf-viewer.umd.js:29:70) 
     at createClass (/app/node_modules/@angular/core/bundles/core.umd.js:12519:20) 
     at createDirectiveInstance (/app/node_modules/@angular/core/bundles/core.umd.js:12364:37) 
     at createViewNodes (/app/node_modules/@angular/core/bundles/core.umd.js:13822:53) 
     at Object.createEmbeddedView (/app/node_modules/@angular/core/bundles/core.umd.js:13699:5) 
     at TemplateRef_.createEmbeddedView (/app/node_modules/@angular/core/bundles/core.umd.js:11774:38) 
     at ViewContainerRef_.createEmbeddedView (/app/node_modules/@angular/core/bundles/core.umd.js:11486:52) 
     at NgIf._updateView (/app/node_modules/@angular/common/bundles/common.umd.js:2887:45) 
     at NgIf.set [as ngIf] (/app/node_modules/@angular/common/bundles/common.umd.js:2843:18) 
     at updateProp (/app/node_modules/@angular/core/bundles/core.umd.js:12699:37) 
     at checkAndUpdateDirectiveInline (/app/node_modules/@angular/core/bundles/core.umd.js:12406:19) 
     at checkAndUpdateNodeInline (/app/node_modules/@angular/core/bundles/core.umd.js:13973:20) 
     at checkAndUpdateNode (/app/node_modules/@angular/core/bundles/core.umd.js:13916:16) 
     at prodCheckAndUpdateNode (/app/node_modules/@angular/core/bundles/core.umd.js:14640:5) 
     at Object.eval [as updateDirectives] (ng:///DocumentModule/ReadComponent.ngfactory.js:335:5) 
     at Object.updateDirectives (/app/node_modules/@angular/core/bundles/core.umd.js:14362:29) 
     at checkAndUpdateView (/app/node_modules/@angular/core/bundles/core.umd.js:13882:14) 
     at callViewAction (/app/node_modules/@angular/core/bundles/core.umd.js:14233:21) 
     at execEmbeddedViewsAction (/app/node_modules/@angular/core/bundles/core.umd.js:14191:17) 
     at checkAndUpdateView (/app/node_modules/@angular/core/bundles/core.umd.js:13883:5) 
     at callViewAction (/app/node_modules/@angular/core/bundles/core.umd.js:14233:21) 
     at execComponentViewsAction (/app/node_modules/@angular/core/bundles/core.umd.js:14165:13) 
     at checkAndUpdateView (/app/node_modules/@angular/core/bundles/core.umd.js:13888:5) 
     at callViewAction (/app/node_modules/@angular/core/bundles/core.umd.js:14233:21) 
     at execEmbeddedViewsAction (/app/node_modules/@angular/core/bundles/core.umd.js:14191:17) 
     at checkAndUpdateView (/app/node_modules/@angular/core/bundles/core.umd.js:13883:5) 
     at callViewAction (/app/node_modules/@angular/core/bundles/core.umd.js:14233:21) 
     at execComponentViewsAction (/app/node_modules/@angular/core/bundles/core.umd.js:14165:13) 
     at checkAndUpdateView (/app/node_modules/@angular/core/bundles/core.umd.js:13888:5) 
     at callViewAction (/app/node_modules/@angular/core/bundles/core.umd.js:14233:21) 
     at execEmbeddedViewsAction (/app/node_modules/@angular/core/bundles/core.umd.js:14191:17) 
     at checkAndUpdateView (/app/node_modules/@angular/core/bundles/core.umd.js:13883:5) 
     at callViewAction (/app/node_modules/@angular/core/bundles/core.umd.js:14233:21) 
     at execComponentViewsAction (/app/node_modules/@angular/core/bundles/core.umd.js:14165:13) 
     at Object.checkAndUpdateView (/app/node_modules/@angular/core/bundles/core.umd.js:13888:5) 
     at ViewRef_.detectChanges (/app/node_modules/@angular/core/bundles/core.umd.js:11657:22) 
     at /app/node_modules/@angular/core/bundles/core.umd.js:5956:63 
     at Array.forEach (<anonymous>) 
     at ApplicationRef.tick (/app/node_modules/@angular/core/bundles/core.umd.js:5956:25) 
     at /app/node_modules/@angular/core/bundles/core.umd.js:5789:105 
```

**Comment by VadimDez** [comment]
Is there any boilerplate project where I can reproduce this? That would help a lot.

**Comment by steambros** [comment]
@VadimDez Looks like `PDFJS` is undefined when using server side rendering:
https://github.com/VadimDez/ng2-pdf-viewer/blob/c49b6db78da554ac1ffe327a4da258740406f472/src/app/pdf-viewer/pdf-viewer.component.ts#L11-L23

So here
https://github.com/VadimDez/ng2-pdf-viewer/blob/c49b6db78da554ac1ffe327a4da258740406f472/src/app/pdf-viewer/pdf-viewer.component.ts#L51
`PDFJS` is undefined and throws an `Cannot read property 'version' of undefined` error.

**Comment by VadimDez** [comment]
Fixed in version `5.2.1`
